src/Components/Breakaway.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react'
+import Tile from './Tile';
+import TileContainer from './data/TileContainer.js'
+import '../Styles/Tile.css';
+import '../Styles/Breakaway.css';
+
+const Breakaway = () => {
+  let breakCount = 0;
+
+  const breakTile = () => {
+    breakCount += 1;
+  }
+
+  return (
+    <div>
+      {breakCount < 3 && <div className="breakaway" id="breakawayTile"></div>}
```

Comment:
I like what you are doing here.  Is there any way we can pull logic out of the return statement?  Do you think we could have at least one separate file that could handle most of our tile logic, then import a logic block to it's corresponding tile?